pkg/codegen/go/gen_program_expressions.go

```diff
@@ -367,6 +340,10 @@ func (g *generator) GenFunctionCallExpression(w io.Writer, expr *model.FunctionC
 	case "cwd":
 		g.Fgen(w, "func(cwd string, err error) string { if err != nil { panic(err) }; return cwd }(os.Getwd())")
 	default:
+		// toJSON and readDir are reduced away, shouldn't see them here
+		reducedFunctions := codegen.NewStringSet("toJSON", "readDir")
+		contract.Assertf(!reducedFunctions.Has(expr.Name), "unlowered function %s", expr.Name)
+		// TODO: implement "element", "entries", "lookup", "split" and "range"
```

Comment:
Moved the specific cases for elements, entries, lookup etc. to the default case so that it doesn't add more uncovered lines of code, same for toJSON and readDir